lib/std/crypto/25519/ed25519.zig

```diff
@@ -113,6 +113,75 @@ pub const Ed25519 = struct {
             return error.InvalidSignature;
         } else |_| {}
     }
+
+    pub const BatchElement = struct {
+        sig: [signature_length]u8,
+        msg: []const u8,
+        public_key: [public_length]u8,
+    };
+
+    pub fn verify_batch(comptime count: usize, signature_batch: [count]BatchElement) !void {
```

Comment:
should be `camelCase`?

Reply:
Oh yes, it should be `camelCase`. Lemme change this right away.